src/Microsoft.DotNet.Build.CloudTestTasks/PackageFiles/CloudTest.targets

```diff
@@ -375,18 +381,30 @@
         <ResultsUriWSAS>$(ResultsWriteOnlyToken)</ResultsUriWSAS>
         <Product>$(TestProduct)</Product>
         <Architecture>$(Platform)</Architecture>
-        <Configuration>$(ConfigurationGroup)$(ConfigurationSuffix)</Configuration>
         <BuildNumber>$(BuildMoniker)</BuildNumber>
         <Branch>$(Branch)</Branch>
-      </BuildComplete>
+      </BuildCompleteTemplate>
+      <BuildCompleteTemplateV2 Include="%(TestListFile.BuildCompleteJson)">
+        <DropContainerSAS>$(DropUriReadOnlyToken)</DropContainerSAS>
+        <ListUri>$(DropUri)%(TestListFile.Filename)%(TestListFile.Extension)$(DropUriReadOnlyToken)</ListUri>
+        <QueueId>$(TargetQueue)</QueueId>
+        <ResultsUri>$(ResultsUri)/%(TestListFile.CorrelationId)</ResultsUri>
+        <ResultsUriRSAS>$(ResultsReadOnlyToken)</ResultsUriRSAS>
+        <ResultsUriWSAS>$(ResultsWriteOnlyToken)</ResultsUriWSAS>
+        <Build Condition="'$(IsOfficial)'=='true'">$(OfficialBuildId)</Build>
+        <Build Condition="'$(Build)'==''">$(CurrentDate)</Build>
+        <Type>$(HelixJobType)</Type>
+        <Source Condition="'$(IsOfficial)'!=''">official/$(TestProduct)/$(Branch)</Source>
+        <Source Condition="'$(HelixSource)'==''">pr/$(TestProduct)/$(Branch)</Source>
+        <Source Condition="'$(HelixSource)'!=''">pr/$(HelixSource)</Source>
```

Comment:
If someone has specifically set the HelixSource, we should just accept it (without prepending pr), since presumably they know what they were doing.